js/artists.js

```diff
@@ -1,119 +1,153 @@
-        imgSrc: './images/aphex.jpg'
-    },
-    {
-        name: 'The Strokes',
-        occupations: 'Rock Band',
-        desc: `The Strokes are an American rock band formed in New York City in 1998.
+    imgSrc: './images/aphex.jpg'
+  },
+  {
+    name: 'The Strokes',
+    occupations: 'Rock Band',
+    desc: `The Strokes are an American rock band formed in New York City in 1998.
         The band is composed of lead singer and songwriter Julian Casablancas, guitarists
         Nick Valensi and Albert Hammond Jr., bassist Nikolai Fraiture, and drummer Fabrizio Moretti.
         They were a leading group of the early-2000s indie rock revival.`,
-        imgSrc: './images/thestrokes.jpg'
-    },
-    {
-        name: 'Jungle',
-        occupations: 'Electronic Band',
-        desc: `Jungle is a British electronic music project founded in 2013 by producers,
+    imgSrc: './images/thestrokes.jpg'
+  },
+  {
+    name: 'Jungle',
+    occupations: 'Electronic Band',
+    desc: `Jungle is a British electronic music project founded in 2013 by producers,
         Josh Lloyd-Watson and Tom McFarland, who are based in London.`,
-        imgSrc: './images/jungle.jpg'
-    },
-    {
-        name: 'Dermot Kennedy',
-        occupations: 'Singer, songwriter',
-        desc: `Dermot Joseph Kennedy is an Irish singer-songwriter. He is best known for his
+    imgSrc: './images/jungle.jpg'
+  },
+  {
+    name: 'Dermot Kennedy',
+    occupations: 'Singer, songwriter',
+    desc: `Dermot Joseph Kennedy is an Irish singer-songwriter. He is best known for his
         2019 singles "Outnumbered" and "Power Over Me", his 2020 single "Giants", and his feature
         on the 2020 Meduza single "Paradise".`,
-        imgSrc: './images/dermot.jpg'
-    },
-    {
-        name: 'Haim',
-        occupations: 'Rock Band',
-        desc: `Haim is an American rock band, based in Los Angeles and comprising three sisters,
+    imgSrc: './images/dermot.jpg'
+  },
+  {
+    name: 'Haim',
+    occupations: 'Rock Band',
+    desc: `Haim is an American rock band, based in Los Angeles and comprising three sisters,
         Este (bass guitar and vocals), Danielle (lead vocals, guitar, and drums), and Alana Haim
         (guitars, keyboards, and vocals).`,
-        imgSrc: './images/haim.jpg'
-    }
+    imgSrc: './images/haim.jpg'
+  }
 ];
 
 // Functions
 
 function createMoreBtn() {
-    const moreBtn = document.createElement('button');
-    moreBtn.innerText = 'MORE';
-    moreBtn.id = 'more-btn';
-    moreBtn.addEventListener('click', () => {
-        document.querySelectorAll('.artist-card').forEach((item) => {
-            if (!item.classList.contains('visible')) {
-                item.classList.add('visible');
-            }
-        });
-        moreBtn.remove();
+  const moreBtn = document.createElement('button');
+  moreBtn.innerHTML = 'MORE<i id="more-icon" class="fa-solid fa-chevron-down"></i>';
```

Comment:
- [ OPTIONAL]I can see that you did a great work implementing the see more button but there should be some improvement, then click on the see more button, it opens then disappeared means we are not able to close it again, kindly fix this  :point_down: 
![Screenshot from 2023-07-06 21-58-19](https://github.com/Movgto/capstone_1_project/assets/88894525/01b6cec3-2e51-4ca6-8639-4679ab2e4d62)